docs/Usage.md

```diff
@@ -14,9 +14,9 @@ The following example is a React component that simply maintains a Form Builder
 
 ```react
 import React, { Component } from 'react';
- 
```

Comment:
Looks like there was some trailing whitespace in this file. My editor automatically removes it :)